handlers/main.yml

```diff
@@ -1 +1,6 @@
 ---
+
+- name: icinga2_master reload icinga2
+  service:
+    name: icinga2.service
```

Comment:
Shouldn't we use only `icinga2` as the service name thus it's independent from systemd?